scripts/maxmind-db-downloader.js

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var path = require('path');
+
+var async = require('async');
+var CronJob = require('cron').CronJob;
+var DEFAULTS = require(path.join('..', 'lib', 'defaults'));
```

Comment:
As @jrgm pointed out elsewhere, `require` is a special flower and doesn't need `path.join`. My mistake.